### DISCUSSION TITLE
Script wont run "you are missing the required tools to catch this fish"

### DISCUSSION THREAD

**Discussion by AlbinCode** [chsami/Microbot#452] [General]
When i wield my dragon harpoon I get this message when i start the script FishingPlugin, ive tried to make a few changes in the code but still gets this message, is there somehow you know how to implement that function to start the script without having the dragon harpoon in the inventory(having it equipped instead)?

**Comment by yairtu** [comment]
added dragon harpoon support
here is the pr:
https://github.com/chsami/microbot/pull/455
